packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep

```diff
@@ -24,6 +25,6 @@ resource botWebAppSettings 'Microsoft.Web/sites/config@2021-01-15' = {
     M365_CLIENT_SECRET: m365ClientSecret
     M365_TENANT_ID: m365TenantId
     M365_APPLICATION_ID_URI: m365ApplicationIdUri
-    IDENTITY_ID: provisionOutputs.identityOutput.value.identityClientId
+    IDENTITY_ID: 
```

Comment:
seems the expected output is not correct?

Reply:
later pr fix it.